snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/id/assigner/SourceConceptNamespaceAndModuleAssigner.java

```diff
@@ -37,29 +38,41 @@
 public final class SourceConceptNamespaceAndModuleAssigner implements SnomedNamespaceAndModuleAssigner {
```

Comment:
I would rename this assigner to something else that reflects its current functionality better.